.changeset/@graphql-hive_gateway-373-dependencies.md

```diff
@@ -0,0 +1,7 @@
+---
+'@graphql-hive/gateway': patch
```

Comment:
@ardatan should we also have a changeset that describes the changes?

Reply:
Done!